frontend/src/components/maps/hooks/useActiveFeatureLayer.tsx

```diff
@@ -97,6 +103,27 @@ const useActiveFeatureLayer = ({
     }
   }, [selectedProperty]);
 
+  /**
+   * If there is a draft property on the map, attempt to retrieve the corresponding parcel. If we find matching parcel data, use that to draw the active parcel.
+   * Note: currently this is limited to finding one parent in the case of a building. in the future, we may need to find/display all matching parcels.
+   */
+  useDeepCompareEffect(() => {
+    const draftProperty = draftProperties[0];
+    const highlightSelectedProperty = async () => {
+      const parcelLayerData = await parcelsService.findOneWhereContains({
+        lat: draftProperty.geometry.coordinates[1] || 0,
+        lng: draftProperty.geometry.coordinates[0] || 0,
+      } as LatLng);
+      if (parcelLayerData?.features?.length > 0) {
+        activeFeatureLayer?.addData(parcelLayerData.features[0]);
+      }
+    };
+    if (!!activeFeatureLayer) {
+      activeFeatureLayer.clearLayers();
+      highlightSelectedProperty();
+    }
+  }, [draftProperties]);
```

Comment:
I think `activeFeatureLayer`  should be added to the dependency list

Reply:
fair, I'll try that out.